tools/cake_version

```diff
@@ -0,0 +1 @@
+0.29.0
```

Comment:
Outdated version: 0.30.0

Reply:
0.29.0 was the most recent version when I created the project, but sure, I can update to 0.30.0.